js/src/modal.js

```diff
@@ -197,7 +198,21 @@ class Modal {
   dispose() {
     $.removeData(this._element, DATA_KEY)
 
-    $(window, document, this._element, this._backdrop).off(EVENT_KEY)
+    /**
+     * Bellow statement is not a valid
+     * $(window, document, this._element, ...).off(EVENT_KEY)
+     * jQuery does not support this type of constructor
+     * eg: https://codepen.io/iamandrewluca/pen/yRWpoV
+     */
+    const htmlElements = [window, this._element, this._dialog]
+    htmlElements.forEach((htmlElement) => $(htmlElement).off(EVENT_KEY))
```

Comment:
```js
[window, this._element, this._dialog].forEach((el) => $(el).off(EVENT_KEY))

Reply:
this will not work, example
![image](https://user-images.githubusercontent.com/1881266/47725375-1e1dad80-dc61-11e8-8ff4-652140f5ff5c.png)